vhdbuilder/packer/convert-sig-to-classic-storage-account-blob.sh

```diff
@@ -67,8 +68,9 @@ echo "Converted $sig_resource_id to $disk_resource_id"
 # shellcheck disable=SC2102
 sas=$(az disk grant-access --ids $disk_resource_id --duration-in-seconds 3600 --query [accessSas] -o tsv)
 
-echo "Logging into azcopy with MSI"
-azcopy-preview login --identity
+echo "Using azcopy with pool identity: $POOL_IDENTITY_RESOURCE_ID"
+export AZCOPY_AUTO_LOGIN_TYPE="MSI"
```

Comment:
So now we don't need to login at all, azcopy will use the existing creds?

Reply:
we don't need the login command when delegating to a managed identity: https://github.com/Azure/azure-storage-azcopy/wiki/Improved-login-support-for-AzCopy-commands-(with-in-memory-secret-store)#using-a-managed-identity

Reply:
thats pretty neat!